microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -176,69 +199,68 @@ The DOM tree is not the HTML source tagging.
 In short, the **DOM Tree** `!==` HTML.
 -->
 
-### Change HTML from the Sources tool
+###### Change HTML from the Sources tool
 
-If you want to save a change to the HTML of the webpage, use the **Sources** tool.
+If you want to save a change to the webpage HTML, use the **Sources** tool.
 
-1. Navigate to the **Sources** tool.
+1. Click the **Sources** tab.
 
-1. In the **Navigator** pane (on the left), click the **Page** tab.
+1. In the **Navigator** pane, click the **Page** tab beside **Filesystem**. If the **Page** tab isn't showing, click `>>` (More tabs) and then select **Page**.
 
 1. Click **(index)**.  The HTML for the page opens.
 
-1. Replace `<h1>Workspaces Demo</h1>` with `<h1>I ❤️  Cake</h1>`.  Review the following figure.
+1. Replace `<h1>Workspaces Demo</h1>` with `<h1>I Love Cake</h1>`.  The demo title changes.
 
 1. Press `Ctrl`+`S` (Windows, Linux) or `Command`+`S` (macOS) to save the change.
 
 1. Refresh the page.  The `<h1>` element continues to display the new text after the page is refreshed.
 
    ![Change HTML from the Sources tool.](../media/workspaces-workspaces-demo-sources-page-h1.msft.png)
 
-1. Open `~/Desktop/app/index.html`.  The `<h1>` element contains the new text.
+1. Open `~/Desktop/app/index.html` in a text editor.  The `<h1>` element contains the new text.
 
 
 <!-- ====================================================================== -->
-## Step 4: Save a JavaScript change to disk
+#### Save a JavaScript change to disk
```

Comment:
```suggestion
## Save a JavaScript change to disk
```
Seems like a major step, d/k why demoting to h4 which makes it vanish from "In this article" nav in upper right.

Definitely some minor headings at top need demoting to h4, but this is a major section step.